app/javascript/angular/code/services/google/_map.js

```diff
@@ -2,6 +2,8 @@ import _ from "underscore";
 import { buildCustomMarker } from "./custom_marker";
 import MarkerClusterer from "@google/markerclustererplus";
 
+window.__markers = [];
```

Comment:
Using window as a tool to decouple stuff. At some point we will be able to remove things from there